testsuite/integration/Makefile

```diff
@@ -0,0 +1,67 @@
+#
+# Copyright 2022 Hewlett Packard Enterprise Development LP
+# Other additional copyright holders may be indicated within.
+#
+# The entirety of this work is licensed under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+SHELL = /usr/bin/env bash -o pipefail
+.SHELLFLAGS = -ec
+
+SKIP_KIND_GENERATION?=""
+
+.PHONY: setup-kind
+setup-kind:
+	@{ \
+		set -e ; \
+		source kind/kind.sh ; \
+		test ! -z "$(SKIP_KIND_GENERATION)" && echo "Skipping cluster generation" || generate_cluster ; \
+		install_dependencies ; \
+		prep_kubeconfig ; \
+	}
+	cd slurm && docker compose build && docker compose up --wait
```

Comment:
This "compose build" fails for me.  It looks like it's expecting to find a Dockerfile in the slurm/ dir, but you don't have one in there.

```
: dean-mbp; cd testsuite/integration/slurm 
: dean-mbp; ls
README.md		docker-compose.yml	jobs/
: dean-mbp; docker compose build
[+] Building 0.0s (1/2)                                                         
 => [internal] load build definition from Dockerfile                       0.0s
 => => transferring dockerfile: 2B                                         0.0s
failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount3097822521/Dockerfile: no such file or directory
```

Reply:
> This "compose build" fails for me. It looks like it's expecting to find a Dockerfile in the slurm/ dir, but you don't have one in there.
> 
> ```
> : dean-mbp; cd testsuite/integration/slurm 
> : dean-mbp; ls
> README.md		docker-compose.yml	jobs/
> : dean-mbp; docker compose build
> [+] Building 0.0s (1/2)                                                         
>  => [internal] load build definition from Dockerfile                       0.0s
>  => => transferring dockerfile: 2B                                         0.0s
> failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to read dockerfile: open /var/lib/docker/tmp/buildkit-mount3097822521/Dockerfile: no such file or directory
> ```

Never mind.  This workarea was a fresh git-clone, and I had not done the submodule init.
It's fine.